components/chips.tsx

```diff
@@ -0,0 +1,21 @@
+import Link from 'next/link';
+
+export const Chips = ({ items }: { items: string[] }) => {
```

Comment:
Added type to avoid `any`

Reply:
All of these types changes you added for components parameters can be added to a new separate  PR to the upstream vickonrails repo - then our repo doesn't diverge as much

Reply:
created a PR in upstream
- https://github.com/vickonrails/next-starter-peacock/pull/25